Kwartracker/Scenes/Sign-in Page/Content/SignInContentView.swift

```diff
@@ -19,6 +19,8 @@ struct SignInContentView: View {
     private let snsOrTopMargin: CGFloat = 5
     private let footNoteTopMargin: CGFloat = 25
     
+    @EnvironmentObject var store: AppStore
```

Comment:
Put this at first before the normal variables
https://github.com/jaimejazarenoiii/kwartracker-ios/pull/4#discussion_r615573741